recipes/xfinity/privacy.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url":"http://my.xfinity.com/privacy/2015-10",
+  "url":"http://my.xfinity.com/privacy/2016-04/",
```

Comment:
I just verified that our [request package](https://www.npmjs.com/package/sync-request) already follows redirects so this url can be simply /privacy.